sleap_io/io/video.py

```diff
@@ -189,6 +192,18 @@ def frames(self) -> int:
         """Number of frames in the video."""
         return self.shape[0]
 
+    @property
+    def frame_rate(self) -> Optional[float]:
+        """Frames per second of the video."""
+        video_extensions = ["mp4", "avi", "mov", "mj2", "mkv"]
+        if not any(self.filename.endswith(ext) for ext in video_extensions):
```

Comment:
This conditional is necessary because `ImageVideo` inherits from `VideoBackend`. If we try to check the frame rate of `ImageVideo` objects, which are stored as image files, we will get an error.